flood_adapt/api/events.py

```diff
@@ -31,11 +28,11 @@ def get_event(name: str) -> IEvent:
 
 
 def get_event_mode(name: str) -> str:
-    filename = Database().events.get_database_path() / f"{name}" / f"{name}.toml"
+    filename = Database().events.input_path / f"{name}" / f"{name}.toml"
     return Event.get_mode(filename)
 
 
-def create_synthetic_event(attrs: dict[str, Any]) -> ISynthetic:
+def create_synthetic_event(attrs: dict[str, Any]) -> Synthetic:
```

Comment:
Now that this changed from ISynthetic to Synthetic I get see this message from PyLAnce:
Type "IObject[Unknown]" is not assignable to return type "Synthetic"
  "IObject[Unknown]" is not assignable to "Synthetic

Reply:
Dont like these warnings either, added an issue for fixing them #564